doc/5.reference/receive-help.pd

```diff
@@ -17,12 +16,22 @@
 receives are named to tell them whom to connect to. They work across
 windows too. Also \, you can use message boxes as shown:;
 #X msg 84 233 \; help-rcv1 34 \; help-rcv2 67;
-#X obj 172 320 r;
+#X obj 177 324 r;
 #X obj 21 10 receive;
 #X text 79 10 -- receive messages without patch cords;
+#X text 222 235 If invoked without an argument \, "send" creates an
```

Comment:
do you mean "receive" instead?

Reply:
sure, I had copied from [send]'s help file but didn't change it, haha